collections.sh

```diff
@@ -1,14 +1,55 @@
 #!/bin/bash
 set -e
 
-docker_publish()
+
+ecr_create()
 {
-  echo "START: Publish.sh"
-  sh ./scripts/publish.sh
-  echo "END: Publish.sh"
-  echo "START: trivy_scan.sh"
-  sh ./scripts/trivy_scan.sh
-  echo "END: trivy_scan.sh"
+  ECR_REPO=$1
+
+  GET_RESPONSE_BODY=$( curl --location --request GET "$LAZY_API_URL/profiles/prod/regions/$AWS_REGION/ecr/repo" \
```

Comment:
This curl command seems to be wrong. GET shouldn't have a body.

Use this URL <https://lazy.apps.ops-drivevariant.com/redoc#operation/get_repo_policy_profiles__profile__regions__region__ecr_repo__repo_name__repo_policy_get> and see if it fails. If it does, then repo doesn't exist.

A sample implementation will look like

curl -f -L "url" || { wget --method POST -O -- "url" }

Can you also check with @naveen-variant on how to use wget to get the error output to display?

Reply:
Changed and added --content-on-error on wget.